ApplicationLibCode/ProjectDataModel/RimEclipseContourMapView.cpp

```diff
@@ -602,3 +612,19 @@ void RimEclipseContourMapView::zoomAll()
 
     RimEclipseView::zoomAll();
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+bool RimEclipseContourMapView::isFaultLinesVisible() const
+{
+    return meshMode() == RiaDefines::MeshModeType::FAULTS_MESH;
+}
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RimEclipseContourMapView::setFaultLinesVisible( const bool& visible )
```

Comment:
I assume this is a const reference to match the "registerSetMethod". If not is has to change..

Reply:
Correct, this syntax is required.